lib/helpers/answerHelper.js

```diff
@@ -116,12 +116,66 @@ function summaryFormatter(answerObject, fullUiSchema, answerLookup) {
     return answerIndex;
 }
 
+function objectFormatter(obj) {
+    const isArray = Array.isArray(obj);
+    let returnValue = '';
+    if (isArray) {
+        obj.forEach(answer => {
+            // eslint-disable-next-line no-use-before-define
```

Comment:
can you move the `formatAnswerV2` function above this function to avoid the eslint ignore rule

Reply:
Yes, but it's 6 and half. `formatAnswerV2` has a reference to this function so if you move it above this function, you'll need to add this comment to `formatAnswerV2`